### PR TITLE
fix(browser): move fetch to background

### LINKED ISSUE #77: network request blocked by CSP on firefox
currently, network request are being blocked on some websites when using firefox.

CSP-error

### PR DESCRIPTION
**Pull request by maltoze**
this pr fixed two issues:
1. fixes #77 #206 
2. currently, firefox extension is broken because

replace lodash with underscore will solve this.

### CODE REVIEW

File: src/popup/Settings.tsx [hunk 1]
```diff
@@ -1,5 +1,5 @@
 import React, { useCallback, useEffect, useState } from 'react'
-import _ from 'lodash'
+import _ from 'underscore'
```
Comment (yetone) [inline src/popup/Settings.tsx:2]: This is beyond my understanding. Isn't lodash supposed to replace underscore? Has underscore surpassed lodash now? 🤔

Comment (maltoze) [reply src/popup/Settings.tsx:2]: not sure, I haven't found another way yet.😅
related: https://github.com/lodash/lodash/issues/4985

Comment (pengx17) [reply src/popup/Settings.tsx:2]: It looks like this is an overkill. Only equal (deep equal) is being used. We can use some dedicated library like https://www.npmjs.com/package/fast-equals instea.

Comment (yetone) [reply src/popup/Settings.tsx:2]: @pengx17 That makes sense. I'll merge it first on my end and then use this library!

**Comment by yetone** [comment]
LGTM!

**Pull request merged**